CHANGELOG.md

```diff
@@ -0,0 +1,72 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.0.1 - 2018-10-31
```

Comment:
Perhaps during active development, it can be easier to keep the current release by using date, in my opinion. And full semantic versioning might be nice after things converge more.